### ISSUE TITLE
install instructions for EPEL releases should use yum instead of dnf

### ISSUE THREAD

**Issue by tmzullinger** [fedora-infra/bodhi#3017]
The addition of the install instructions (#2799) were deployed with 3.13.0.  They should use `yum` as the command rather than `dnf` for EPEL.  I believe if they use the `--advisory` option they would need to tell users that requires the yum security plugin. I don't use that option, so someone more familiar would need to look at the details.

**Comment by tmzullinger** [comment]
It looks like the `--advisory` option works by default in yum for EL7 (tested on a CentOS-7 container and RHEL-7 VM).  The test I used was:

```
yum update --enablerepo epel-testing --advisory=FEDORA-EPEL-2019-bcf7340f15
```

For EL-6 the option requires `yum-plugin-security`.

Also of note is that `updates-testing` should be `epel-testing` for the EPEL updates template.

**Comment by bowlofeggs** [comment]
Fixed in #3047.
